content/en/security_platform/application_security/getting_started/nodejs.md

```diff
@@ -0,0 +1,103 @@
+---
+title: NodeJS Applications
+kind: documentation
+code_lang: nodejs
+type: multi-code-lang
+code_lang_weight: 50
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-js'
+      tag: 'GitHub'
+      text: 'NodeJS Datadog Library source code'
+---
+
+{{% appsec-getstarted %}}
+
+## Get started
+
+1. **Update or install the latest Datadog NodeJS library package**, at least version 2.2.0, by running:
```

Comment:
You also need to follow the steps for the tracer library first: https://docs-staging.datadoghq.com/kari/appsec-beta-docs/tracing/setup_overview/setup/nodejs/?tab=containers
For users that don't know anything about Datadog's APM library, they will miss the "add `require('dd-trace').init()` to your app" step, AppSec's env var alone won't do anything. It requires the tracer to be set up too.
I'm suggesting either copying their instructions here for a better UX, or pointing to their setup instructions as pre-requisite, for easier maintainability.

Reply:
Looks good! but IMO we should separate the `appsec: true` from this step, for 4 reasons:
- It isn't possible with the command line init
- We'd rather push people to use the env vars (DevOps oriented)
- More clarity IMO
- Users will think they need both the config option and the env var enabled, which will be a source of problems

It would look like this:

2. Import and initialize the library(...)
```javascript
// This line must come before importing any instrumented module.
const tracer = require('dd-trace').init()
```
**(also remove `appsec: true` from the TypeScript tab)**

3. Enable Application Security in your environment by setting the following environment variable **or config option**:

**(revert to the previous step 3 tabs organization, by adding the first "env var" tab and last "config option" tab IIRC)**